packages/ses/src/lockdown.js

```diff
@@ -180,12 +181,20 @@ export const repairIntrinsics = (options = {}) => {
       /** @param {string} debugName */
       debugName => debugName !== '',
     ),
+    regeneratorRuntimeTaming = getenv(
+      'LOCKDOWN_REGENERATOR_RUNTIME_TAMING',
+      'none',
+    ),
```

Comment:
```suggestion
    legacyRegeneratorRuntimeTaming = getenv(
      'LOCKDOWN_LEGACY_REGENERATOR_RUNTIME_TAMING',
      'safe',
    ),
```

Since this is no longer necessary for the current regenerator runtime, I suggest including "legacy" somewhere in the name.

On changing `'none'` to `'safe'`, see comment on line 194 below.